src/screens/create-wallet-confirm-mnemonic/create-wallet-confirm-mnemonic.tsx

```diff
@@ -150,7 +158,10 @@ export const CreateWalletConfirmMnemonicScreenComponent = (
                                 (n: number) => mnemonicsInput[n] === mnemonic[n]
                             );
                             if (valid) {
-                                this.passwordModal.requestPassword().then(password => {
+                                this.passwordModal.requestPassword().then(async password => {
```

Comment:
ai incercat sa pui loading modalu inainte de parola?, va aparea loadingu, se cere parola si cand se inchide parola loadingu e acolo

Reply:
It's not working daca pun loading modal inainte de password. Se blocheaza in Loading, intra Loading din `App.tsx` si Password Modal nu se mai afiseaza